spec/scrum_lint/interactive_linters/card/missing_hash_tag_spec.rb

```diff
@@ -0,0 +1,51 @@
+RSpec.describe ScrumLint::InteractiveLinter::MissingHashTag, '#call' do
+  let(:checker) { described_class.new }
+  let(:mock_reporter) do
+    instance_double(ScrumLint::Reporters::InteractiveReporter)
+  end
+  let(:trello_card) { fake_trello_card(name: '#Eat pineapple') }
+  let(:trello_list) { fake_trello_list(cards: [trello_card]) }
+  let(:trello_board) { fake_trello_board(lists: [trello_list]) }
+  let(:card) do
+    ScrumLint::Trello::CardMapper.(
+      trello_card,
+      list: trello_list,
+      board_name: trello_board.name
+    )
+  end
+
+  before do
+    allow(mock_reporter).to receive(:fail)
+    allow(mock_reporter).to receive(:get_value)
+  end
+
+  it 'returns nil if card already has a hashtag' do
+    result = checker.(card, reporter: mock_reporter)
+
+    expect(result).to be_nil
+    expect(mock_reporter).not_to have_received(:fail)
+  end
+
+  context 'when card does not have a hashtag' do
+    let(:trello_card) { fake_trello_card(name: 'ice cream') }
+
+    it 'shows a message' do
```

Comment:
if this script now adds a hashtag to the trello card, do we still need to show an error message?

Reply:
It's not an error message, it's just a message. the `fail` method is misleading - it basically says the card failed the _linter_, not failed as in "threw an error."

Reply:
ahhh

Reply:
ok cool